keras_nlp/models/backbone.py

```diff
@@ -15,14 +15,49 @@
 from keras_nlp.api_export import keras_nlp_export
 from keras_nlp.backend import config
 from keras_nlp.backend import keras
-from keras_nlp.utils.preset_utils import check_preset_class
+from keras_nlp.utils.preset_utils import check_config_class
+from keras_nlp.utils.preset_utils import get_registered_presets
 from keras_nlp.utils.preset_utils import load_from_preset
 from keras_nlp.utils.python_utils import classproperty
 from keras_nlp.utils.python_utils import format_docstring
 
 
 @keras_nlp_export("keras_nlp.models.Backbone")
 class Backbone(keras.Model):
+    """Base class for all `Backbone` models.
+
+    A `Backbone` is the basic architecture for a given NLP model. Unlike a
+    `keras_nlp.models.Task`, a `Backbone` is not tailored to any specific loss
+    function and training setup. A `Backbone` generally outputs the last hidden
+    states of a architecture before any output predictions.
+
+    A `Backbone` can be used in one of two ways:
```

Comment:
Generally, add line breaks before/after any sort of special markup (the list, in this case). The standard markdown renderer has issues otherwise. The renderer that GitHub runs is highly customized.